hardhat.config.ts

```diff
@@ -11,7 +11,11 @@ export default {
       gas: 12000000,
       blockGasLimit: 0x1fffffffffffff,
       allowUnlimitedContractSize: true,
-      timeout: 1800000
+      timeout: 1800000,
```

Comment:
Could we have two separate networks:
1) hardhat-local : to run unit tests
2) hardhat-fork: to run integration tests (like the one with pangolin)